### ISSUE TITLE
Hi! The network does not work, it is impossible to watch the video from YouTube. What to do?

### ISSUE THREAD

**Issue by andrkulv** [rsodre/VLCSyphon#6]
Hi! The network does not work, it is impossible to watch the video from YouTube. What to do?

**Comment by andrkulv** [comment]
on high sierra 10.13.06

**Comment by rsodre** [comment]
Yes, the network is very unstable, and I have no intention to debug or fix it.

The best approach would be to get the current VLC master and reintroduce Syphon to it. But that's also something I have no current plans to do.

**Comment by andrkulv** [comment]
it's a pity, of course, if this function worked, it would be cool, it's cool to take any streaming video from YouTube without the need to store it on disk and process it in Quatrtz composer or in MAX. So far I have not found a suitable application other than this, but here this feature is deprecated (

**Comment by andrkulv** [comment]
stream processing in real time ;)

**Comment by rsodre** [comment]
I have played YouTube videos on VLCSyphon in the past.
Maybe they changed something on the stream that made this version not recognize anymore, which is usual.
